projects/ngxpert/hot-toast/src/lib/components/hot-toast/hot-toast.component.ts

```diff
@@ -200,18 +201,18 @@ export class HotToastComponent implements OnInit, AfterViewInit, OnDestroy, OnCh
   }
 
   ngOnInit() {
```

Comment:
use useEffect

Reply:
When replacing the ngOnInit method with Effect, toastRef becomes null, resulting in an error.

Reply:
I will check